src/components/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react';
+import { bem } from '../../utils';
+import { Text } from '../Text';
+import styles from './Toggle.scss';
+
+export interface Props extends Omit<React.InputHTMLAttributes<HTMLInputElement>, 'type'> {
+    /** A unique id to reference this toggle */
+    id: string;
+    /** Label related to toggle */
+    children?: string;
+    /** Should be Toggle disabled or not */
+    disabled?: boolean;
+    /** Ref to access the toggle input element */
+    ref?: React.RefObject<HTMLInputElement>;
+}
+
+const { block, elem } = bem('Toggle', styles);
+
+export const Toggle: React.FC<Props> = React.forwardRef((props, ref) => {
+    const { children, disabled, id, ...rest } = props;
+    return (
+        <label {...block(props)} htmlFor={id}>
+            <div {...elem('toggleContainer')}>
+                <input
```

Comment:
Consider adding  `role="switch"` and `aria-checked="true/false"` for accessibility

Reply:
hm, why do we need this if we already have input with type checkbox?
we can switch on/off by Space.
Another question is that we do not see that toggle is focused. But I found the same with Checkbox component, so postponed it for a while to ask you.

Reply:
As far as I can tell it's not exactly the same as checkbox:
>The ARIA switch role is functionally identical to the checkbox role, except that instead of representing "checked" and "unchecked" states, which are fairly generic in meaning, the switch role represents the states "on" and "off."

But it's not super relevant to us.

With regard to focus state: good catch, I'll have a look.

Reply:
Hi @carlobernardini !
> With regard to focus state: good catch, I'll have a look.

Maybe we can do this focusing functionality (if needed) in dedicated story for both: Toggle and Checkbox. What do you think?

Reply:
Sounds good to me 👍